static/js/crowdsource.directives.js

```diff
@@ -96,5 +97,17 @@
                 });
             }
         };
+    }   
+
+    function autoFocus($timeout) {
```

Comment:
@dmorina Auto-focus needs a directive as single page apps do not support auto-focus when shifting to different views as page is not loaded again.